gems/quilt_rails/lib/quilt_rails/react_renderable.rb

```diff
@@ -6,9 +6,10 @@ module Quilt
   module ReactRenderable
     include ReverseProxy::Controller
 
-    def render_react
+    def render_react(headers: {})
       raise DoNotIntegrationTestError if Rails.env.test?
 
+      @headers = headers
```

Comment:
I guess it's sort of the railsy thing to do to  stick this on the class instance rather than pipe it down as a parameter, but it feels a bit weird to me to do it here.

Reply:
I can definitely change this, first week using ruby and rails so still learning the conventions around things like this.